src/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ endif()
 # This matches libtool's usage of DLL_EXPORT
 if(WIN32)
   set_target_properties(secp256k1 PROPERTIES DEFINE_SYMBOL "DLL_EXPORT")
+  target_compile_definitions(secp256k1 INTERFACE $<$<NOT:$<BOOL:${BUILD_SHARED_LIBS}>>:SECP256K1_STATICLIB>)
```

Comment:
I assume this should be `SECP256K1_STATIC` ?

Reply:
The Windows CMake task didn't catch this mistake (https://cirrus-ci.com/task/6074245453709312) because it uses DLLs. Would it make sense to add a CI task that uses static linkage?

Reply:
> The Windows CMake task didn't catch this mistake ([cirrus-ci.com/task/6074245453709312](https://cirrus-ci.com/task/6074245453709312)) because it uses DLLs. Would it make sense to add a CI task that uses static linkage?

The similar idea came to my head :)
[Implemented](https://github.com/bitcoin-core/secp256k1/pull/1367#issuecomment-1614333593).